index.html

```diff
@@ -139,7 +139,75 @@ <h2>What is a Hackathon?</h2>
     </section>
     <section id="schedule">
     </section>
-    <section id="faq">
+    <section id="faq-content">
+        <div id="faq-wrapper">
+            <h1>Frequently Asked Questions</h1>
+            <div id="faq-cards">
+                <div class="card">
+                    <button class="accordion"><i class="fa fa-plus fa-plus-pink" aria-hidden="true"></i>Who can participate in BrickHack?</button>
+                    <div class="panel">
+                        <p>
+                            Anyone currently enrolled as a student can attend! If you don't fit that description, you're
+                            absolutely welcome to attend as a mentor or volunteer. All attendees must be 18 years or older.
+                        </p>
+                    </div>
+                </div>
+
+                <div class="card">
+                    
+                    <button class="accordion"><i class="fa fa-plus fa-plus-pink" aria-hidden="true"></i>Do I need a team to join?</button>
```

Comment:
Maybe space it out a bit more? I'm neutral either way
```suggestion
                    <button class="accordion">
                        <i class="fa fa-plus fa-plus-pink" aria-hidden="true"></i>
                        Do I need a team to join?
                    </button>
```